instrumentation/jsf/jsf-mojarra-3.0/javaagent/build.gradle.kts

```
@@ -0,0 +1,34 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("org.glassfish")
+    module.set("jakarta.faces")
+    versions.set("[3,)")
+    assertInverse.set(true)
+  }
```

Looks like there are several muzzle failures for these two new libraries:
```

> Task :instrumentation:jsf:jsf-myfaces-3.0:javaagent:muzzle-AssertPass-org.apache.myfaces.core-myfaces-impl-3.0.0 FAILED
FAILED MUZZLE VALIDATION: io.opentelemetry.javaagent.instrumentation.myfaces.v3_0.MyFacesInstrumentationModule mismatches:
-- io.opentelemetry.javaagent.instrumentation.jsf.jakarta.JsfRequest:33 Missing class jakarta.el.MethodExpression
-- io.opentelemetry.javaagent.instrumentation.myfaces.v3_0.MyFacesErrorCauseExtractor:16 Missing class jakarta.el.ELException
-- io.opentelemetry.javaagent.instrumentation.jsf.jakarta.JsfRequest:31 Missing class jakarta.el.MethodExpression
```

Perhaps you need to add the `el-api` dep to muzzle:
```suggestion
    extraDependency("jakarta.el:jakarta.el-api:4.0.0")
  }
```

---

Yeah, that line seems to be needed 😄 

Unfortunately, I don't know how all these dependencies and muzzle are working under the hood, and just trying different things. After we replaced `compileOnly` with `library`, lastestDepTests fail. Seems like all the `library` dependencies are removed (I was expecting it overrides with redefined only).

I thought that muzzle is a kind of condition when the instrumentation should be loaded, and the dependency on `faces` implementation should be enough. This extra dependency seems redundant, and the `el-api` theoretically should be included in the `jsf-jakarta-common:testing` and loaded whenever the package is used, because it's common for myfaces and mojarra. But all my attempts to move it results in failed tests.